modules/openxr/scene/openxr_composition_layer_cylinder.cpp

```diff
@@ -52,9 +52,11 @@ OpenXRCompositionLayerCylinder::OpenXRCompositionLayerCylinder() {
 		aspect_ratio, // aspectRatio
 	};
 	openxr_layer_provider = memnew(OpenXRViewportCompositionLayerProvider((XrCompositionLayerBaseHeader *)&composition_layer));
+	XRServer::get_singleton()->connect("reference_frame_changed", callable_mp(this, &OpenXRCompositionLayerCylinder::update_transform));
 }
 
 OpenXRCompositionLayerCylinder::~OpenXRCompositionLayerCylinder() {
+	XRServer::get_singleton()->disconnect("reference_frame_changed", callable_mp(this, &OpenXRCompositionLayerCylinder::update_transform));
```

Comment:
```suggestion
```
Unnecessary, signals are disconnected on destruction, unless you've had problems with this